cms/djangoapps/contentstore/views/item.py

```diff
@@ -492,8 +510,32 @@ def _create_item(request):
             definition_data=data,
             metadata=metadata,
             runtime=parent.runtime,
+            position=child_position
         )
 
+        # Entrance Exams: Grader assignment
+        if settings.FEATURES.get('ENTRANCE_EXAMS', False):
+            course = store.get_course(usage_key.course_key)
+            if hasattr(course, 'entrance_exam_enabled') and course.entrance_exam_enabled:
+                if category == 'sequential' and request.json.get('parent_locator') == course.entrance_exam_id:
+                    grader = {
+                        "type": "Entrance Exam",
+                        "min_count": 0,
+                        "drop_count": 0,
+                        "short_label": "Entrance",
+                        "weight": 0
+                    }
+                    grading_model = CourseGradingModel.update_grader_from_json(
+                        course.id,
+                        grader,
+                        request.user
+                    )
+                    CourseGradingModel.update_section_grader_type(
+                        created_block,
+                        grading_model['type'],
+                        request.user
+                    )
+
```

Comment:
seems like the setting of the grading policy should be outside of _create_item at a slightly higher level of the business logic. Likewise, if we delete an entrance exam, we'd want to make sure we remove this grader policy.

Also, does this new grading policy show up in Studio's grading setting page (sorry, I can't remember the exact name of the page), as we wouldn't want authors to edit/delete it.